client/Index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import axios from 'axios';
+import MainPhotoCard from './components/PhotoCards/MainPhotoCard.jsx';
+
+class Index extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photos: null,
+    };
+  }
```

Comment:
jw what your reasons are for creating a whole component for main photo?  is this just for styled.components?

Reply:
I've actually switched my logic and and won't be using cards because it doesn't make sense for what i'm trying to do. Thanks for asking :)